mtp_api/settings/base.py

```diff
@@ -270,6 +265,14 @@ def callback_add_role_name(envelope):
         },
     },
 }
+if APPLICATIONINSIGHTS_CONNECTION_STRING:
+    # Sends messages from `mtp` logger to Azure Application Insights
+    LOGGING['handlers']['azure'] = {
```

Comment:
Neat!

Reply:
probs better to just edit the "default" settings rather than duplicate?